airbyte-webapp/scripts/load-declarative-schema.sh

```diff
@@ -0,0 +1,26 @@
+set -e
```

Comment:
Having a comment at the top of this script explaining why we have it may be helpful for anyone coming here from just seeing the script used in `generate-client` in package.json